server/src/main/java/org/apache/druid/server/initialization/ServerConfig.java

```diff
@@ -100,6 +103,10 @@ public ServerConfig()
   @Min(1)
   private long maxScatterGatherBytes = Long.MAX_VALUE;
 
+  @JsonProperty
+  @Min(1)
+  private int maxSubqueryRows = 100000;
```

Comment:
this should probably be added to the documentation

Reply:
Oh yeah, good point.

Reply:
I'm hoping to update all the docs at once in a future patch